RecoTracker/TrackProducer/test/BuildFile.xml

```diff
@@ -26,13 +28,11 @@
 
 <library file="FakeCPEFiller.cc" name="FakeCPEFiller">
   <use name="SimTracker/TrackerHitAssociation"/>
-  <use name="RecoTracker/TransientTrackingRecHit"/>
   <flags EDM_PLUGIN="1"/>
 </library>
 
 <environment>
   <bin file="testMuonTrackRefitting.cpp">
     <flags TEST_RUNNER_ARGS="/bin/bash RecoTracker/TrackProducer/test testMuonTrackRefitting.sh"/>
-    <use name="FWCore/Utilities"/>
```

Comment:
This has already been taken care of by https://github.com/cms-sw/cmssw/pull/36227/

Reply:
Ahh so that's why I had merge conflicts, I see.

Reply:
this still looks to be here.

Reply:
I dont see it
https://github.com/cms-sw/cmssw/blob/d5383bb0177451c6a25e9ce71a220044f36335de/RecoTracker/TrackProducer/test/BuildFile.xml

Reply:
Ahh finally I understood what you meant, sorry, I did it now.